SPECS/atop/format.patch

```diff
@@ -0,0 +1,33 @@
+--- drawbar.c~	2023-05-13 03:20:27.000000000 -0500
```

Comment:
Did we spin this patch ourselves? It looks like this was fixed [upstream](https://github.com/Atoptool/atop/commit/957ff648436fa4a6f08ad9a8c5ea856a5f33ef5b) perhaps? 

Incidentally, it also looks like v2.10.0 came out this week and includes the above linked change.

Reply:
Yes a new version is up but it requires glib2 which we do not ship.
Should we add glib2 as well to move to the latest version ?

Reply:
The `glib.spec` in CBL-Mariner provides `glib2`. Is there a specific version of glib2 that's required?

Reply:
Yes, thank you. It seem that upgrade to 2.10.0 is non-trivial, for now I will merge this and circle back when I get some additional cycles.